iati/tests/data/strings.py

```diff
@@ -4,11 +4,74 @@
+LONG_TEXT = (
```

Comment:
@andylolz Everything else looks fine but could I get your thoughts on the changes to this file in particular? In team we try to avoid mid-sentence hard-wrapping where possible in favour of locally soft-wrapping text and excluding deliberately long strings from linting on a case by case basis. Interested to get your views on why you've chosen this style here as it doesn't upset my local linters so I'm unaware of any potential benefits of this change?

Reply:
Yeah, so essentially I had no idea how to fix this nicely, and this was the best I could manage.

You can see on travis that it’s flake8 complaining about this:
https://travis-ci.org/IATI/preview-website/jobs/401877458

> iati/tests/data/strings.py:7:256: E501 line too long (3274 > 255 characters)
> iati/tests/data/strings.py:11:256: E501 line too long (800 > 255 characters)

I’m more than happy to revert d1fa7f3, and resolve the errors in whatever is the preferred way.